internal/distributed/indexnode/service.go

```diff
@@ -211,9 +211,6 @@ func (s *Server) start() error {
 func (s *Server) Stop() error {
 	Params := &paramtable.Get().IndexNodeGrpcServerCfg
 	log.Debug("IndexNode stop", zap.String("Address", Params.GetAddress()))
-	if s.indexnode != nil {
```

Comment:
the indexnode stop step likes querynode,  it should execute s.indexnode.Stop() first, then close grpc server.